pkg/ovsdb/client.go

```diff
@@ -190,7 +190,9 @@ func Dial(addressList [][]string, initialize func(*OVSDB) error, options map[str
 
 				conn, err = tls.Dial("tcp", address, cfg)
 			} else {
-				conn, err = net.Dial(network, address)
+				// DialNet is platform dependent. On windows platform, the "named pipe" connection is supported
+				// and the corresponding network name is "winpipe".
+				conn, err = DialNet(network, address)
```

Comment:
There is quite advanced logic for dialing within client.go inside Dial function.
For example there is already conditional statement for scenario when 'network' variable has value 'ssl'.
I would prefer to extend that logic there unless there is something specific I could be missing why the code was taken outside of that function.

Reply:
It's because the "winpipe" is only support by windows platform. So we implement DiaNet in different files for different platform to avoid compile issue:
- dia.go: For linux, darwin
- dia_windows.go: For windows

Reply:
Ok, tested. You can't compile on linux with
import (
	"github.com/Microsoft/go-winio"
)
That's why you need remove unnecessary file before compilation.